app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -10,13 +10,19 @@ accessibility_criteria: |
   * be keyboard focusable
 examples:
   default:
+    data:
+      label:
+        text: "What is your name?"
+      name: "name"
+  email_type:
     data:
       label:
         text: "What is your email address?"
+        type: "email"
```

Comment:
Looks like the type might be in the wrong place here?

Reply:
Doh! good catch

Reply:
@nickcolley should be good now.